src/main/groovy/com/adarshr/gradle/testlogger/TestResultWrapper.groovy

```diff
@@ -21,7 +21,8 @@ class TestResultWrapper {
     boolean isLoggable() {
         testLoggerExtension.showPassed && testResult.successfulTestCount ||
             testLoggerExtension.showSkipped && (testResult.resultType == SKIPPED || testResult.skippedTestCount) ||
-            testLoggerExtension.showFailed && testResult.failedTestCount
+            testLoggerExtension.showFailed && testResult.failedTestCount ||
+            testLoggerExtension.showOnlySlow && (isTooSlow() || isMediumSlow())
```

Comment:
There is something wrong with this expression.

Reply:
I was able to get it to work by changing the method body to be as below. If you can do it in a more readable manner, please do so.

```
          if (testLoggerExtension.showOnlySlow && (!isTooSlow() || !isMediumSlow())) {
              return false
          }
 
          testLoggerExtension.showPassed && testResult.successfulTestCount ||
              testLoggerExtension.showSkipped && (testResult.resultType == SKIPPED || testResult.skippedTestCount) ||
              testLoggerExtension.showFailed && testResult.failedTestCount
```

Reply:
Refactored this method a little bit to make it more readable